### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by RuoyuChen10** [open-mmlab/mmdetection#9389]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmdetection

### Environment

Python: 3.8.12 (default, Oct 12 2021, 13:49:34) [GCC 7.5.0]
CUDA available: True
GPU 0: NVIDIA GeForce RTX 3090
CUDA_HOME: :/usr/local/cuda-11.6:/usr/local/cuda-11.6
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.12.1+cu116
PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201402
  - Intel(R) oneAPI Math Kernel Library Version 2021.4-Product Build 20210904 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.6.0 (Git Hash 52b5f107dd9cf10910aaa19cb47f3abf9b349815)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.6
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.3.2  (built against CUDA 11.5)
  - Magma 2.6.1
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.6, CUDNN_VERSION=8.3.2, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -fabi-version=11 -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.12.1, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.13.1+cu116
OpenCV: 4.5.5
MMCV: 1.7.0
MMCV Compiler: GCC 9.3
MMCV CUDA Compiler: 11.6
MMDetection: 2.26.0+31c8495

### Reproduces the problem - code sample

```python
python tools/train.py configs/faster_rcnn/faster_rcnn_r50_caffe_c4_1x_coco.py
```

### Reproduces the problem - command or script

```python
python tools/train.py configs/faster_rcnn/faster_rcnn_r50_caffe_c4_1x_coco.py
```

### Reproduces the problem - error message

```shell
2022-11-27 04:02:21,605 - mmdet - INFO - Epoch [1][1350/58633]  lr: 2.000e-02, eta: 1 day, 16:29:11, time: 0.091, data_time: 0.007, memory: 7525, loss_rpn_cls: nan, loss_rpn_bbox: nan, loss_cls: nan, acc: 24.6757, loss_bbox: nan, loss: nan
```

### Additional information

I tried to use faster r-cnn to train the ms coco data set. Using official documents, I changed the learning rate from 0.02 to 0.0002, and did not load the pre-trained model. However, at the very beginning, the model crashed, and the predicted classification results were all Background class or class 0. What could be the reason for this? I checked the MS COCO dataset, the picture is complete, and the annotation is normal, but the training seems to have no effect.

**Comment by BIGWangYuDong** [comment]
Hi, sorry for the late reply. The default Batch size is 16 with lr 0.02. If your GPUs is not 16, try to use `--auto-lr` to set the lr smaller. Moreover, you can follow FAQ https://github.com/open-mmlab/mmdetection/blob/master/docs/en/faq.md#training

**Comment by BIGWangYuDong** [comment]
`--auto-scale-lr` will help you enable automatically scaling lr

**Comment by RuoyuChen10** [comment]
> `--auto-scale-lr` will help you enable automatically scaling lr

Thank you for your reply. I have successfully debugged the 8-card 2080ti. In addition, it should be noted that if fpn is not used, the feature graph generated by backbone is more than 50 in length and width, which can produce better results.
